app/decorators/suscribir/suscripcion_decorator.rb

```diff
@@ -0,0 +1,13 @@
+module Suscribir
+  class SuscripcionDecorator < Draper::Decorator
+    delegate_all
+
+    def url_desuscribir
+      h.engine_suscribir.pedir_confirmacion_baja_url(id, token)
```

Comment:
It's a reference to how the engine routes are declared on the main app:
```ruby
  mount Suscribir::Engine => '/suscribir', as: 'engine_suscribir'
```
I've also tried without `engine_routes.`, thinking that inside the engine it wasn't needed, but it doesn't work...:-(